dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -688,6 +688,12 @@
         # This setting can also be set using the environment variable ZEEBE_BROKER_EXPERIMENTAL_ROCKSDB_DISABLEWAL
         # disableWal: false
 
+      consistencyChecks:
```

Comment:
:x: Please check indentation. It looks to be on the same level as `rocksDB` (see also comment on variable name). If it is supposed to be under rocksDB, then RocksDB would have to be commented in

Also this could be commented out altogether since it only shows what the default value is.